cli/cmd/resources/managers.go

```diff
@@ -10,12 +10,12 @@ import (
 // set apiKey to nil for no-op.
 // set to empty string for "no api key" (non odigos cloud mode).
 // set to a valid api key for odigos cloud mode.
-func CreateResourceManagers(client *kube.Client, odigosNs string, odigosTier common.OdigosTier, proTierToken *string, config *common.OdigosConfiguration, odigosVersion string) []resourcemanager.ResourceManager {
+func CreateResourceManagers(client *kube.Client, odigosNs string, odigosTier common.OdigosTier, proTierToken *string, config *common.OdigosConfiguration, odigosVersion, installationMethod string) []resourcemanager.ResourceManager {
```

Comment:
nit: consider using `K8sInstallationMethod` as the type for new argument to communicate that this is an enum with close set of values, and only convert to string when populating the config map

```suggestion
func CreateResourceManagers(client *kube.Client, odigosNs string, odigosTier common.OdigosTier, proTierToken *string, config *common.OdigosConfiguration, odigosVersion, installationMethod K8sInstallationMethod) []resourcemanager.ResourceManager {
```